L1Trigger/L1CaloTrigger/plugins/L1EGammaCrystalsEmulatorProducer.cc

```diff
@@ -595,28 +602,35 @@ void L1EGCrystalClusterEmulatorProducer::produce(edm::Event& iEvent, const edm::
               }
               if (abs(hit.dieta(centerhit)) <= 2 && abs(hit.diphi(centerhit)) <= 2) {
                 e5x5 += hit.energy();
+                n5x5++;
```

Comment:
What are these variables you're adding used for?

Reply:
These are originally from https://github.com/cms-l1t-offline/cmssw/pull/1069

Reply:
Okay. Then I think I'm done here